bigtable/internal/option/option.go

```diff
@@ -201,3 +202,103 @@ func EnableBigtableConnectionPool() bool {
 	}
 	return enableBigtableConnPool
 }
+
+// Logf logs the given message to the given logger, or the standard logger if
+// the given logger is nil.
+func logf(logger *log.Logger, format string, v ...interface{}) {
+	if logger == nil {
+		log.Printf(format, v...)
+	} else {
+		logger.Printf(format, v...)
+	}
+}
+
+var debug = os.Getenv("CBT_ENABLE_DEBUG") == "true"
+
+// Debugf logs the given message *only if* the global Debug flag is true.
+// It reuses Logf to handle the nil logger logic and prepends "DEBUG: "
+// to the message.
+func Debugf(logger *log.Logger, format string, v ...interface{}) {
```

Comment:
it is weird to see logging functionality in the `option` package. More like util or maybe just include in the connpool file?